tests/decorators/decorators.go

```diff
@@ -108,4 +108,10 @@ var (
 	NoFlakeCheck = Label("no-flake-check")
 	// FlakeCheck decorates tests that are dedicated to the check-tests-for-flakes test lane.
 	FlakeCheck = Label("flake-check")
+
+	/* Potentially disruptive tests */
+
+	// HighStoragePoolRequired indicates that the test may fail in a cluster with a low storage pool capacity.
+	// This decorator can be used to skip the test as the failure might not indicate a functional problem.
+	HighStoragePoolRequired = Label("high-storage-pool-required")
```

Comment:
naming is hard but maybe extralargestoragepoolrequired?

Reply:
Yeah, I know.. hmm maybe just `disruptive`? It's really not about the pool, it could be huge and still fails if the provisioner rounds up aggresively

Reply:
Isn't `Disruptive` already a tag to skip tests? Wouldn't it be confusing to have both? If I understand correctly, tests using `Disruptive` will always be skipped.

Reply:
Yeah whole thing is weird lets not go in that rabbit hole for now. LargeStoragePoolrequired sounds good

Reply:
:+1: ack